lib/ProductOpener/DataQualityFood.pm

```diff
@@ -597,7 +597,14 @@ sub check_nutrition_data_energy_computation ($product_ref) {
 			my ($ignore_energy_calculated_error, $category_id)
 				= get_inherited_property_from_categories_tags($product_ref, "ignore_energy_calculated_error:en");
 
-			if (not((defined $ignore_energy_calculated_error) and ($ignore_energy_calculated_error eq 'yes'))) {
+			if (
+				(
+					not((defined $ignore_energy_calculated_error) and ($ignore_energy_calculated_error eq 'yes'))
+					and (  ($product_ref->{nutriments}{"energy-kj_value"} > 55)
```

Comment:
Can you add a comment to explain why we have this condition on energy? (so that we know if we revisit it later...)